lib/import-export-visitor.js

```diff
@@ -225,15 +224,28 @@ class ImportExportVisitor extends Visitor {
       }, "declaration");
 
     } else {
-      // Otherwise, since the exported value is an expression, it's
-      // important that we wrap it with parentheses, in case it's something
-      // like a comma-separated sequence expression.
-      overwrite(this, decl.start, dd.start, exportDefaultPrefix);
+      // Otherwise, since the exported value is an expression, we use the
+      // special module.exportDefault(value) form.
 
       path.call(this.visitWithoutReset, "declaration");
       assert.strictEqual(decl.declaration, dd);
 
-      overwrite(this, dd.end, decl.end, exportDefaultSuffix, true);
+      let prefix = "module.exportDefault(";
+      let suffix = ");";
+
+      if (dd.type === "SequenceExpression") {
+        // If the exported expression is a comma-separated sequence
+        // expression, this.code.slice(dd.start, dd.end) may not include
+        // the vital parentheses, so we should wrap the expression with
+        // parentheses to make absolutely sure it is treated as a single
+        // argument to the module.exportDefault method, rather than as
+        // multiple arguments.
+        prefix += "(";
+        suffix = ")" + suffix;
+      }
```

Comment:
We [do](https://github.com/benjamn/reify/tree/b84bd5695a34059b1580281e29b5c610a0ed3e79/test/output/default-expression)! Previously, we were just wrapping the argument with parentheses unconditionally, but now we try to avoid it except in this one very strange case.